scripts/release/generate-changelog.js

```diff
@@ -118,6 +127,25 @@ ${internalChanges.join('\n')}
 `.replace(/\(#(\d+)\)/gm, (_, id) => `([#${id}](https://github.com/DataDog/browser-sdk/pull/${id}))`)
 }
 
+function getAffectedPackages(hash) {
+  const changedFiles = command`git diff-tree --no-commit-id --name-only -r ${hash}`.run().trim().split('\n')
+  const affectedPackages = new Set()
+
+  changedFiles.forEach((file) => {
+    if (file.startsWith('packages/rum-core')) {
+      affectedPackages.add('rum')
+      affectedPackages.add('rum-slim')
+    }
+    PACKAGES.forEach((pkg) => {
+      if (file.startsWith(`packages/${pkg}`)) {
+        affectedPackages.add(pkg)
+      }
+    })
+  })
```

Comment:
❓Question: Is it possible for us to check the dependencies programmatically with package json files or sth? 

We might have more packages on the way of bundle size reduction. It would be super convenient if it's all automatic.

Reply:
great idea ! I just changed that